### PR TITLE
Split Forms in Controllers

### LINKED ISSUE #49: make a new directory "forms"
currently we have a file forms.py which should be split for different uses e.g users(login & register), courses etc. So for that we shall have a new directory with those files and import them where a new import is needed.

### PR DESCRIPTION
**Pull request by LiorRey**
Resolved #49

The file forms.py had been split to different forms file in a new 'forms' directory under 'controllers'.
Each class (AddCourse, LoginForm, RegistrationForm, ResourceForm) got a new form file.

### CODE REVIEW

**Review by mosesnagar — CHANGES_REQUESTED** [review]
The CI is broken, and you also should resolve the conflicts.

File: tutor/controllers/forms/registration_form.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+from wtforms import StringField, PasswordField, SubmitField
+from wtforms.validators import DataRequired, Length, EqualTo
+
+
+class RegistrationForm(FlaskForm):
+    username = StringField('Username', validators=[DataRequired(), Length(min=2, max=20)])
+    email = StringField('Email', validators=[DataRequired(), Length(min=2, max=20)])
+    password = PasswordField('Password', validators=[DataRequired(), Length(min=3, max=15)])
+    confirm_password = PasswordField('Confirm Password', validators=[DataRequired(), EqualTo('password')])
+    submit = SubmitField('Sign Up')
```
Comment (golan13) [inline tutor/controllers/forms/registration_form.py:11]: You are missing two functions that validate the username and email to make sure there aren't duplicate usernames or emails. this is why it's failing the tests
`def validate_username` and `def validate_email`

**Review by golan13 — CHANGES_REQUESTED** [review]

File: tutor/controllers/forms/registration_form.py (hunk 1 above)
Comment (LiorRey) [reply tutor/controllers/forms/registration_form.py:12]: Added those validations - now it's fixed!

**Review by mosesnagar — APPROVED** [review]
Maybe write a description for that on `CONTRIBUTING.md` file, to let everyone know about that new way. I think you can also open a new issue for that with a new PR.

**Review by golan13 — APPROVED** [review]

**Comment by LiorRey** [comment]
> Maybe write a description for that on `CONTRIBUTING.md` file, to let everyone know about that new way. I think you can also open a new issue for that with a new PR.

Done.

**Pull request merged**